gensim/models/keyedvectors.py

```diff
@@ -500,6 +500,7 @@ def most_similar(self, positive=None, negative=None, topn=10, restrict_vocab=Non
             List of words that contribute negatively.
         topn : int, optional
             Number of top-N similar words to return.
+        last: Only for set-Based analogy. The word which belongs in the same pair as the expected word. 
```

Comment:
see doc for another parameters, should be
```
param_name : type
    Description
```

Reply:
Fixed